templates/components/form/fields_macros.html.twig

```diff
@@ -263,10 +263,18 @@
       {% set options = {'required': true}|merge(options) %}
    {% endif %}
 
+   {% if options.checkIsExpired %}
+      {% if value < "now"|date('Y-m-d H:i:s') %}
```

Comment:
`value` will be a date here, may be a date format should be applied to ensure comparison is done correctly.
```suggestion
      {% if value|date('Y-m-d H:i:s') < "now"|date('Y-m-d H:i:s') %}
```